tests/vertexonly/test_vertex_only_mesh_generation.py

```diff
@@ -148,7 +148,7 @@ def test_generate_cell_midpoints(parentmesh):
         else:
             cell_num = parentmesh.locate_cell(out_of_mesh_point)  # should return None
         if cell_num is not None:
-            assert all(f.dat.data_ro[cell_num] == vm.coordinates.dat.data_ro[i])
+            assert (f.dat.data_ro[cell_num] == vm.coordinates.dat.data_ro[i]).all()
```

Comment:
is this preferred style then?

Reply:
when the shape is 1D then the `all(...)` version doesn't work because you get a single numpy.bool back (which is not iterable), but does implement `.all()`